### PR TITLE
[627] Galleries filter

### PR DESCRIPTION
**Pull request by mmiedlarz**
trello card:
https://trello.com/c/8sfV3sdq/627-ios-when-viewing-a-user-profile-i-want-to-see-only-that-users-galleries

### CODE REVIEW

File: CreatubblesAPIClient/Sources/Requests/Gallery/MyGalleriesRequest.swift [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 import UIKit
 
-enum MyGalleriesRequestFilter {
+public enum GalleriesRequestFilter {
     case none
```
Comment (JakubJankowski) [inline CreatubblesAPIClient/Sources/Requests/Gallery/MyGalleriesRequest.swift:29]: Does it make sense to actually have a `none` filter? If a request isn't filtered, then shouldn't the filter be simply nil? (I see that that's exactly how it works)
Maybe .unknown would make more sense?

File: CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift [hunk 2]
```diff
@@ -49,7 +49,53 @@ class GalleriesResponseHandlerSpec: QuickSpec {
...
+                waitUntil(timeout: TestConfiguration.timeoutMedium) {
+                    done in
+                    _ = sender.login(TestConfiguration.username, password: TestConfiguration.password) {
+                        (error: Error?) -> Void in
+                        expect(error).to(beNil())
+                        _ = sender.send(request, withResponseHandler:GalleriesResponseHandler {
+                            (galleries: Array<Gallery>?, pageInfo: PagingInfo?, error: Error?) -> Void in
+                            expect(galleries).notTo(beNil())
+                            expect(error).to(beNil())
+                            expect(pageInfo).notTo(beNil())
```
Comment (JakubJankowski) [inline CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift:68]: We're not testing if returned galleries indeed belong to the user. Maybe something like

`expect(galleries.first.owner.identifier).equals(userId)`

but for all galleries?

**Review by JakubJankowski — APPROVED** [review]
Looks good, two minor comments.

File: CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift (hunk 2 above)
Comment (mmiedlarz) [reply CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift:68]: Good point, will add it

File: CreatubblesAPIClient/Sources/Requests/Gallery/MyGalleriesRequest.swift (hunk 1 above)
Comment (mmiedlarz) [reply CreatubblesAPIClient/Sources/Requests/Gallery/MyGalleriesRequest.swift:29]: hmm I think I added it due to objc functionality, but will see what can be done here :)

**Review by loregr — APPROVED** [review]
code: 👍 
func: n/a

**Pull request merged**